package.json

```diff
@@ -8,7 +8,7 @@
   ],
   "devDependencies": {
     "lerna": "^3.22.1",
-    "@types/yeoman-generator": "^3.0.2"
+    "@types/yeoman-generator": "^5.2.7"
```

Comment:
Why put this on in the root level package.json? Should it be in "cli/package.json"
Also, shouldn't ideally the @types package version match the version of the yeoman-generator?

Reply:
setting it to each sub-package.json to the following version
"@types/yeoman-generator": "^4.11.0",